bellatrix.web/src/main/java/solutions/bellatrix/web/infrastructure/ByImageBase64.java

```diff
@@ -0,0 +1,28 @@
+package solutions.bellatrix.web.infrastructure;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class ByImageBase64 extends By {
+    private final String base64EncodedImage;
+
+    public ByImageBase64(String base64EncodedImage) {
+        this.base64EncodedImage = base64EncodedImage;
+    }
+
+    public static By byImageBase64(String encodedImage) {
+        return new ByImageBase64(encodedImage);
+    }
+
+    @Override
+    public List<WebElement> findElements(SearchContext context) {
+        return null;
+    }
+
+    public String toString() {
+        return "By.imageBase64: " + this.base64EncodedImage;
+    }
```

Comment:
I am thinking of renaming the class to ByImage, which has overloads for base64 string, and also file path.

Reply:
Another option is to create it as a separate class